packages/next/compiled/@next/react-dev-overlay/client.js

```diff
@@ -4,4 +4,4 @@
ht-white);\n    background-color: var(--color-ansi-red);\n    box-shadow: 0px var(--size-gap-double) var(--size-gap-quad)\n      rgba(0, 0, 0, 0.25);\n  }\n"],["\n  [data-nextjs-toast] {\n    position: fixed;\n    bottom: var(--size-gap-double);\n    left: var(--size-gap-double);\n    max-width: 420px;\n    z-index: 9000;\n  }\n\n  @media (max-width: 440px) {\n    [data-nextjs-toast] {\n      max-width: 90vw;\n      left: 5vw;\n    }\n  }\n\n  [data-nextjs-toast-wrapper] {\n    padding: 16px;\n    border-radius: var(--size-gap-half);\n    font-weight: 500;\n    color: var(--color-ansi-bright-white);\n    background-color: var(--color-ansi-red);\n    box-shadow: 0px var(--size-gap-double) var(--size-gap-quad)\n      rgba(0, 0, 0, 0.25);\n  }\n"])));n.styles=i;var o},71:function(e,n,t){"use strict";var r=this&&this.__makeTemplateObject||function(e,n){if(Object.defineProperty){Object.defineProperty(e,"raw",{value:n})}else{e.raw=n}return e};var a=this&&this.__createBinding||(Object.create?function(e,n,t,r){if(r===undefined)r=t;Object.defineProperty(e,r,{enumerable:true,get:function(){return n[t]}})}:function(e,n,t,r){if(r===undefined)r=t;e[r]=n[t]});var i=this&&this.__setModuleDefault||(Object.create?function(e,n){Object.defineProperty(e,"default",{enumerable:true,value:n})}:function(e,n){e["default"]=n});var o=this&&this.__importStar||function(e){if(e&&e.__esModule)return e;var n={};if(e!=null)for(var t in e)if(t!=="default"&&Object.prototype.hasOwnProperty.call(e,t))a(n,e,t);i(n,e);return n};n.__esModule=true;n.styles=n.BuildError=void 0;var l=o(t(522));var s=t(824);var u=t(382);var c=t(504);var f=t(373);var d=function BuildError(e){var n=e.message;var t=l.useCallback((function(){}),[]);return l.createElement(u.Overlay,{fixed:true},l.createElement(s.Dialog,{type:"error","aria-labelledby":"nextjs__container_build_error_label","aria-describedby":"nextjs__container_build_error_desc",onClose:t},l.createElement(s.DialogContent,null,l.createElement(s.DialogHeader,{className:"nextjs-container-build-error-header"},l.createElement("h4",{id:"nextjs__container_build_error_label"},"Failed to compile")),l.createElement(s.DialogBody,{className:"nextjs-container-build-error-body"},l.createElement(c.Terminal,{content:n}),l.createElement("footer",null,l.createElement("p",{id:"nextjs__container_build_error_desc"},l.createElement("small",null,"This error occurred during the build process and can only be dismissed by fixing the error.")))))))};n.BuildError=d;n.styles=(0,f.noop)(p||(p=r(["\n  .nextjs-container-build-error-header > h4 {\n    line-height: 1.5;\n    margin: 0;\n    padding: 0;\n  }\n\n  .nextjs-container-build-error-body footer {\n    margin-top: var(--size-gap);\n  }\n  .nextjs-container-build-error-body footer p {\n    margin: 0;\n  }\n\n  .nextjs-container-build-error-body small {\n    color: #757575;\n  }\n"],["\n  .nextjs-container-build-error-header > h4 {\n    line-height: 1.5;\n    margin: 0;\n    padding: 0;\n  }\n\n  .nextjs-container-build-erro
```

Comment:
You may want to check out past commits. I'm not sure if we're supposed to change the emitted JS code. Typically these are only updated before a new version release.

Reply:
I am also unsure if this code should be in the pull request, however it seems that a separate PR created by the Next.js team also changes the emitted JS code: https://github.com/vercel/next.js/pull/33868. It is probably best to ask the team directly when we open this PR in the Next.js repository.